analyzers/tests/ITs.JsonParser.Test/StringAssertionsExtensions.cs

```diff
@@ -0,0 +1,15 @@
+﻿using FluentAssertions.Primitives;
+
+namespace ITs.JsonParser.Test;
+
+public static class StringAssertionsExtensions
```

Comment:
Stole this from the other project, because this project is .NET8 and the other one is .NET7-windows, so I can't target it.

Reply:
You should be able to target `net8-windows`, no?